taxonomyServices/src/main/java/org/sahsu/taxonomyservices/claMLTaxonomyService.java

```diff
@@ -0,0 +1,78 @@
+package org.sahsu.taxonomyservices;
+
```

Comment:
I don't know how you got this to build. The file has a lowercase 'C' at the start and an uppercase 'L'. It needs to be called `ClaMlTaxonomyService.java`, to match the class name

Reply:
Yes, I noticed this in master (and fixed it). I think it built because at one point there were two versions on my machine. This appears to be a git issue - the rename failed. I have now renamed it on my machine and fixed the repository - which had two versions on it.